zoninator.php

```diff
@@ -1382,7 +1382,10 @@ private function filter_zone_feed_fields( $results ) {
 		$i = 0;
 		foreach ( $results as $result ) {
 			foreach( $whitelisted_fields as $field ) {
-					$filtered_results[$i]->$field = $result->$field;
+				if ( ! is_object( $filtered_results[$i] ) ) {
```

Comment:
We should probably instantiate `$filtered_results` as an array before the loops.